drivers/cime/ensemble_driver.F90

```diff
@@ -261,6 +261,10 @@ subroutine SetModelServices(ensemble_driver, rc)
           endif
           call shr_file_setLogUnit (logunit)
 
+          ! Disable hierarchy connectors
```

Comment:
remove commented code

Reply:
my initial intention was keeping it because it might help for debugging in the future but as you suggest, i removed it